packages/framework/src/servers/nest.ts

```diff
@@ -1,3 +1,17 @@
+/*
+ * Re-export all top level exports from the main package.
+ * This results in better DX reduces the chances of the dual package hazard for ESM + CJS packages.
+ *
+ * Example:
+ *
+ * import { serve, Client, type Workflow } from '@novu/framework/express';
+ *
+ * instead of
+ *
+ * import { serve } from '@novu/framework/express';
+ * import { Client, type Workflow } from '@novu/framework';
```

Comment:
```suggestion
 * import { NovuModule, Client, type Workflow } from '@novu/framework/nest';
 *
 * instead of
 *
 * import { serve } from '@novu/framework/nest';
 * import { Client, type Workflow } from '@novu/framework';
```